<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu::CMenuGroup</class>
 <widget class="QWidget" name="MainMenu::CMenuGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="CPanelFrame" name="menuWidget" native="true"/>
   </item>
   <item>
    <widget class="CPanelFrame" name="contentWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPanelFrame</class>
   <extends>QWidget</extends>
   <header>MainMenu/Include/PanelFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
